uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/Panel.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use CRM_HRContactActionsMenu_Component_Group as ActionsGroup;
+
+/**
+ * Class CRM_HRContactActionsMenu_Component_Panel
+ */
+class CRM_HRContactActionsMenu_Component_Panel {
```

Comment:
I was thinking about the classes name and and the organization and then I realized we probably made a confusion here.  This is not really a panel, but the component itself. Panels are actually what we are calling now "main" and "highlighted" areas. I suggest we rename this class to `CRM_HRContactActionsMenu_Component_Menu` as it is the component that represents the entire menu